lib/app/utils/force_update.dart

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'dart:io';
+
+import 'package:ion/app/exceptions/exceptions.dart';
+import 'package:ion/app/features/core/providers/env_provider.c.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class ForceUpdateUtil {
+  ForceUpdateUtil._();
+
+  static Future<void> handleForceUpdateRedirect() async {
+    if (Platform.isAndroid) {
+      const appId = EnvVariable.ION_ANDROID_APP_ID;
+      final androidUrl = 'https://play.google.com/store/apps/details?id=$appId';
+      await _openUrl(androidUrl);
+    } else if (Platform.isIOS) {
+      const appId = EnvVariable.ION_IOS_APP_ID;
+      final iosUrl = 'https://apps.apple.com/app/id$appId';
+      await _openUrl(iosUrl);
+    } else if (Platform.isWindows || Platform.isLinux || Platform.isMacOS) {
+      await _openWebsite();
+    }
+  }
+
+  static Future<void> _openUrl(String url) async {
+    final uri = Uri.parse(url);
+    if (await canLaunchUrl(uri)) {
+      await launchUrl(uri, mode: LaunchMode.externalApplication);
+    } else {
+      throw ForceUpdateCouldntLaunchUrlException(url: url);
+    }
+  }
+
+  static Future<void> _openWebsite() async {
+    const fallbackWebsite = 'https://example.com'; //TODO: Replace with the actual website
+    await _openUrl(fallbackWebsite);
+  }
+
+  static bool isVersionOutdated(String localVersion, String remoteVersion) {
+    final localParts = localVersion.split('.').map(int.parse).toList();
+    final remoteParts = remoteVersion.split('.').map(int.parse).toList();
+
+    for (var i = 0; i < remoteParts.length; i++) {
+      if (localParts.length <= i || localParts[i] < remoteParts[i]) {
+        return true;
+      } else if (localParts[i] > remoteParts[i]) {
+        return false;
+      }
+    }
+    return false;
+  }
```

Comment:
i would like to do not use `static` methods as much as possible because at future we ll not write unit tests for these. does it makes sense for you ?